src/resolvers-cassandra/Messages/queries.js

```diff
@@ -29,6 +35,49 @@ function byEdges(args, res) { // eslint-disable-line no-unused-vars
  * @returns {Promise.<Feature>}
  */
 function event(args, res) { // eslint-disable-line no-unused-vars
+  const eventById = 'SELECT * FROM fortis.events WHERE id = ?';
+
+  return new Promise((resolve, reject) => {
+    const eventId = args && args.messageId;
+    if (!eventId) {
+      reject('No event id to fetch specified');
+      return;
+    }
+
+    cassandraConnector.executeQuery(eventById, [eventId])
+    .then(rows => {
+      if (rows.length > 1) {
+        reject(`Got more ${rows.length} events with id ${eventId}`);
+        return;
+      }
+
+      const ev = rows[0];
+      featureServiceClient.fetchById(ev.detectedplaceids || [])
+      .then(features => {
+        resolve({
+          type: ev.pipeline,
+          coordinates: features.map(feature => feature.bbox),
+          properties: {
+            edges: ev.detectedkeywords,
+            messageid: ev.externalid,
+            createdtime: ev.event_time,
+            sentiment: ev.computedfeatures && ev.computedfeatures.sentiment &&
+              ev.computedfeatures.sentiment.pos_avg > ev.computedfeatures.sentiment.neg_avg
+              ? ev.computedfeatures.sentiment.pos_avg - ev.computedfeatures.sentiment.neg_avg + 0.6
+              : ev.computedfeatures.sentiment.neg_avg - ev.computedfeatures.sentence.pos_avg,
+            title: ev.title,
+            originalSources: ev.pipeline &&
+              [ev.pipeline],
+            language: ev.eventlangcode,
+            source: ev.sourceurl,
+            fullText: ev.messagebody
+          }
+        });
+      })
+      .catch(err => reject(err));
```

Comment:
I think you could do `catch(reject);` too right? The way you have it now is more explicit, so I like that way, too.